### ISSUE TITLE
Listed as extraneous when checking installed packages 

### ISSUE THREAD

**Issue by chocobuckle** [olehkuchuk/inline-css#2]
Because the package is installed with 'npm i inline-css-cli' but then in the package.json script the command to run it is 'inline-css' so NPM thinks 'inline-css-cli' is not being used and lists it as extraneous. Not the end of the world, but a little annoying.

**Comment by hzlmn** [comment]
Yes it is a case. I took such name because `inline-css` was taken. If you have any name suggestion, feel free to push me and I will change it.

**Comment by chocobuckle** [comment]
@hzlmn Well is it possible to just change the script command used in 'package.json' files to 'inline-css-cli', instead of 'inline-css'? Then NPM wouldn't list it as extraneous, right?

**Comment by hzlmn** [comment]
Yes, it's possible if this name works for you I will update it. I took `inline-css` because it's more convenient. But maybe you are right as anyone can simply add shell alias.

**Comment by chocobuckle** [comment]
@hzlmn Yeah I think "inline-css-cli' as the node script command would make things less confusing, or else someone might accidentally prune it when they see it incorrectly listed as extraneous. It's a great, simple to use package by the way - thanks for your work! :)
